docs/components/bottom_navigation.md

```diff
@@ -0,0 +1,293 @@
+| **Icon** | `icon` |
+| **Color** | `unselectedItemColor` |
+| **Size** | `size` on `unselectedIconTheme` |
+
+
+
+**Inactive text label attributes**
+
+| &nbsp; | Property |
+| --- | --- |
+| **Text label** | `title` |
+| **Color** | `unselectedItemColor` |
+| **Disabled Text Color** | N/A |
+| **Typography** | `unselectedLabelStyle` |
+
+**Active icon attributes**
+
+| &nbsp; | Property |
+| --- | --- |
+| **Icon** | `icon` |
+| **Color** | `selectedItemColor` |
+| **Size** | `size` on `selectedIconTheme` |
+
+
+
+**Active text label attributes**
+
+| &nbsp; | Property |
+| --- | --- |
+| **Text label** | `title`|
+| **Color** | `selectedItemColor` |
+| **Disabled Text Color** | N/A |
+| **Typography** | `selectedLabelStyle` |
+
+## Theming a bottom nav bar
+
+The following example shows a bottom navigation bar with the [Material Shrine Theme](https://material.io/design/material-studies/shrine.html).
+
+* [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/bottom_navigation/theme)
+
+![FABs with Shrine theming](assets/bottom_navigation/flutter-bottom-nav-themed.png)
+
+```
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: _buildShrineTheme(),
+      title: 'Flutter Demo',
+      home: MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  int _currentIndex = 0;
+
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    final textTheme = Theme.of(context).textTheme;
+    return Scaffold(
+      bottomNavigationBar: BottomNavigationBar(
+        type: BottomNavigationBarType.fixed,
+        currentIndex: _currentIndex,
+        backgroundColor: colorScheme.surface,
+        selectedItemColor: colorScheme.onSurface,
+        unselectedItemColor: colorScheme.onSurface.withOpacity(.60),
+        selectedLabelStyle: textTheme.caption,
+        unselectedLabelStyle: textTheme.caption,
+        onTap: (value) {
+          // Respond to item press.
+          setState(() => _currentIndex = value);
+        },
+        items: [
+          BottomNavigationBarItem(
+            title: Text('Favorites'),
+            icon: Icon(Icons.favorite),
+          ),
+          BottomNavigationBarItem(
+            title: Text('Music'),
+            icon: Icon(Icons.music_note),
+          ),
+          BottomNavigationBarItem(
+            title: Text('Places'),
+            icon: Icon(Icons.location_on),
+          ),
+          BottomNavigationBarItem(
+            title: Text('News'),
+            icon: Icon(Icons.library_books),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+ThemeData _buildShrineTheme() {
```

Comment:
Should this be limited to only the params that affect the bottom nav?

Reply:
Done, although I left the color scheme and text theme code's the same as other examples.